doc/man/8/rbd.rst

```diff
@@ -247,6 +247,7 @@ Commands
 
 :command:`deep cp` (*src-image-spec* | *src-snap-spec*) *dest-image-spec*
   Deep copy the content of a src-image into the newly created dest-image.
+  Deep copy will copy all the image snapshots as well as the "head".
   Dest-image will have the same size, object size, image format, and snapshots as src-image.
```

Comment:
It already says that: "... and snapshots as src-image."  That is `cp` doesn't mention snapshots, while `deep cp` does.  

Perhaps it would be better to add a "Note that snapshots aren't copied, use `deep cp` command for that" note to `cp`?

Reply:
@idryomov - thanks for the review. I removed the sentence from `deep cp` description and added note to `cp` description. Does it look better now?